index.bs

```diff
@@ -559,31 +583,56 @@ Width of the play-area bounds in meters. The bounds are defined as an axis-align
 <dfn attribute for="VRStageParameters">sizeZ</dfn>
 Depth of the play-area bounds in meters. The bounds are defined as an axis-aligned rectangle on the floor. The center of the rectangle is at (0,0,0) in standing-space coordinates. These bounds are defined for safety purposes. Content should not require the user to move beyond these bounds; however, it is possible for the user to ignore the bounds resulting in position values outside of this rectangle.
 
-
-## Navigator Interface extension ## {#interface-navigator}
+## VR Interface ## {#interface-vr}
 
 <pre class="idl">
-partial interface Navigator {
-  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getVRDisplays();
-  readonly attribute FrozenArray&lt;VRDisplay&gt; activeVRDisplays;
+interface VR : EventTarget {
+  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getDisplays();
+  readonly attribute FrozenArray&lt;VRDisplay&gt; activeDisplays;
+
```

Comment:
I was suggesting adding `navigator.vr.getAvailability` (as a replacement for the `navigator.vrEnabled` property, which should obviously be removed).

This would respect the [Feature Policy](https://docs.google.com/document/d/1k0Ua-ZWlM_PsFCFdLMa8kaVTo32PeNZ4G7FFHqpFx4E/edit#heading=h.4yubgixv5l6b) (which supports `<iframe enable="vr" src="https://example.com/scene.html"></iframe>` – see issue #86).

Besides the aforementioned `<iframe>` case, there are cases (e.g., browser extension) when a UA may not want to (or cannot attempt to) expose VR capabilities. (A VR display could still be connected, but the page may not be privileged to use it.)

Some similar methods in other web APIs:
* [`new PresentationRequest([urls]).getAvailability().then(…)`](https://www.w3.org/TR/presentation-api/#dom-presentationrequest-getavailability)
* [`navigator.bluetooth.getAvailability().then(…)`](https://webbluetoothcg.github.io/web-bluetooth/#availability)
* [`navigator.usb.requestDevice({filters: [], optionalServices: [], acceptAllDevices: true})`.then(…)](https://wicg.github.io/webusb/#dom-usb-requestdevice)

Alternatively, instead of a `Promise<boolean> getAvailability()`, we could have the following:

```idl
    readonly attribute VRAvailability availability;
    attribute EventHandler onavailabilitychange;
    enum VRAvailability {
        'available',
        'unavailable',
        'unknown'
    };
```